stack.yaml

```diff
@@ -20,7 +20,7 @@ packages:
 
 extra-deps:
 - git: https://github.com/serokell/cardano-sl
-  commit: 2986b304d8b89a14f1fd9be031f626d1c8a45088 # ariadne branch
+  commit: 04a633ac236a20477bfc1e5bc4b8e76b834f9079 # rebased binary-update branch
```

Comment:
Note that you should make a PR to the `ariadne` branch and update the revision here after that PR is merged.
I've described it [here](https://www.notion.so/serokell/Additional-workflow-68044e748c7340f4a418d7ee0292198e).